apiserver/facades/agent/sshconnreq/register.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package sshconnreq
+
+import (
+	"reflect"
+
+	apiservererrors "github.com/juju/juju/apiserver/errors"
+	"github.com/juju/juju/apiserver/facade"
+)
+
+// Register is called to expose a package of facades onto a given registry.
+func Register(registry facade.FacadeRegistry) {
+	registry.MustRegister("SSHConnReqAgent", 1, func(ctx facade.Context) (facade.Facade, error) {
```

Comment:
can we come up with a better facade name - i'm not a big fan of saving a few characters, if we can improve readability.
e.g. SSHConnectionRequestsAgent